tests/backup/backup_kdmp_test.go

```diff
@@ -0,0 +1,721 @@
+			for _, namespace := range bkpNamespaces {
+				pods, err := core.Instance().GetPods(namespace, nil)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("getting pods from namespace [%s] ", namespace))
+				for _, pod := range pods.Items {
+					containerMountPathList := make([]string, 0)
+					containerPaths := schedops.GetContainerPVCMountMap(pod)
+					for containerName, paths := range containerPaths {
+						log.Infof(fmt.Sprintf("containerName:%s,paths: %s ", containerName, paths))
+						containerMountPathList = append(containerMountPathList, paths...)
+					}
+					if len(containerMountPathList) > 0 {
+						podMountPathMap[pod.Name] = containerMountPathList
+						log.Infof(fmt.Sprintf("the list of mountPath within pod [%v] are [%v]", pod.Name, containerMountPathList))
+					}
+					scMountPathMap, err = schedops.GetContainerPVCMountMapWithSC(pod)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("getting storage class and mountpath mapping for pod [%s] ", pod.Name))
+				}
+			}
+		})
+
+		Step("Fetch the existing directories and files within mountPath before writing files and directories", func() {
+			log.InfoD("Fetch the existing directories and files within mountPath before writing files and directories")
+			for _, namespace := range bkpNamespaces {
+				pods, err := core.Instance().GetPods(namespace, nil)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("getting pods from namespace [%s] ", namespace))
+				for _, pod := range pods.Items {
+					if len(podMountPathMap[pod.Name]) > 0 {
+						existingFileDirList := make([]string, 0)
+						for _, mountPath := range podMountPathMap[pod.Name] {
+							log.Infof(fmt.Sprintf("Fetch the existing directories and files within mountPath [%s] before writing files and directories", mountPath))
+							existingFileList, existingDirList, err := FetchFilesAndDirectoriesFromPod(pod, mountPath, nil)
+							existingFileDirList = append(existingFileDirList, existingFileList...)
+							existingFileDirList = append(existingFileDirList, existingDirList...)
+							existingFileDirMountPathMap[mountPath] = existingFileDirList
+							dash.VerifyFatal(err, nil, fmt.Sprintf("fetching files and directory from mountpath [%s] for pod [%s]", mountPath, pod.Name))
+						}
+					}
+				}
+			}
+		})
+
+		Step("Create nested directories and files into container mountPath for applications", func() {
+			log.InfoD("Create nested directories and files into container mountPath for applications")
+			for _, namespace := range bkpNamespaces {
+				pods, err := core.Instance().GetPods(namespace, nil)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("getting pods from namespace [%s] ", namespace))
+				for _, pod := range pods.Items {
+					if len(podMountPathMap[pod.Name]) > 0 {
+						for _, mountPath := range podMountPathMap[pod.Name] {
+							scName := scMountPathMap[mountPath]
+							DirectoryConfig := PodDirectoryConfig{
+								BasePath:          mountPath,
```

Comment:
Please make it 1000 x 1000

Reply:
I'm keeping it to 1 lakh file due to few scale limitation we have.   Currently it will create 10^3 = 1000 directories and 100 files in each directory.